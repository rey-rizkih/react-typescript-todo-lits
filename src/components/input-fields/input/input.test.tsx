import { shallow } from "enzyme";
import { InputStyled } from ".";

describe("<Input/> rendering", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<InputStyled data-testid="input" />);
  });

  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should render one", () => {
    expect(wrapper.find("[data-testid='input']")).toHaveLength(1);
  });
});
