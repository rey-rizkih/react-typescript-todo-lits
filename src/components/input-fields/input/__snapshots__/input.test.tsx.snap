// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input/> rendering renders correctly 1`] = `
<Fragment>
  <Insertion
    cache={
      Object {
        "insert": [Function],
        "inserted": Object {},
        "key": "css",
        "nonce": undefined,
        "registered": Object {},
        "sheet": StyleSheet {
          "_insertTag": [Function],
          "before": null,
          "container": <head />,
          "ctr": 0,
          "insertionPoint": undefined,
          "isSpeedy": false,
          "key": "css",
          "nonce": undefined,
          "prepend": undefined,
          "tags": Array [],
        },
      }
    }
    isStringTag={false}
    serialized={
      Object {
        "map": undefined,
        "name": "ytn2i7",
        "next": undefined,
        "styles": "width:100%;border-radius:50px;padding:20px 30px;font-size:25px;border:none;background-color:white;transition:0.2s;box-shadow:inset 0 0 5px black;font-family:Arial;line-height:normal;:focus-within{box-shadow:0 0 10px 1000px rgba(0, 0, 0, 0.5);outline:none;}.MuiInput-input{padding:0;height:auto;}",
        "toString": [Function],
      }
    }
  />
  <ForwardRef(Input)
    className="css-ytn2i7"
    data-testid="input"
  />
</Fragment>
`;
